src/app/venter/complaint-card/complaint-card.component.css

```diff
@@ -0,0 +1,48 @@
+.complaint-card {
+    text-decoration: none;
+}
+
+.complaint-divider {
+    padding: 1%;
+}
+
+.complaint-card-content {
+    font-size: medium;
+    padding-top: 3%;
+}
+
+.complaint-description {
+    white-space: pre-wrap;
+}
+
+.complaint-location-details {
+    padding-top: 2%;
+}
+
+.card {
+    margin: 5%;
+    width: 80%;
+}
+
+.footer {
+    position: relative;
+    text-align: center;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/wncc/instiapp-angular/pullRequest?prid=5455571)